stage0/src/image_header.rs

```diff
@@ -157,6 +161,17 @@ impl Image {
         header.version
     }
 
+    // TODO(AJS): Replace get_version with this?
```

Comment:
the reason why the hubris `get_version`  doesn't access this directly is because if we're not running with TrustZone (i.e. on an s28) we don't have full access to be able to access the header the way it's currently designed. It probably would be much better to have it access the same data.

Reply:
@labbott Are you saying to just go ahead and do that now in this PR?

Reply:
I don't think it necessarily needs to do it in this PR but we should make sure it happens sooner rather than later to avoid confusion when the two inevitably end up out of sync.

Reply:
Ok. Got it. I'll likely do it in this PR.